dbt/models/shared_columns.md

```diff
@@ -786,6 +786,12 @@ percentage of fair cash value at which a property is assessed for taxing
 purposes. See `ccao.class_dict` for more information
 {% enddocs %}
 
+## is_corner_lot
+
+{% docs shared_column_is_corner_lot %}
+Corner lot indicator.
```

Comment:
**suggestion:** Can you open an issue to improve the docs here (and for the table) describing the algorithm used?

Reply:
Can do.